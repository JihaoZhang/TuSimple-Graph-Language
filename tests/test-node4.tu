int main()
{	
	node @{string} n1;
	node @{string} n2;

	new n1;
	new n2;
	prints(n1.value());
	prints(n2.value());

	n1.setvalue("abc");
	n2.setvalue("def");

	prints(n1.value());
	prints(n2.value());

	prints(n1.name());
	prints(n2.name());

	return 0;
}