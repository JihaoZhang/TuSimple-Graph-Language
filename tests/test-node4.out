

abc
def
n1
n2